src/plugins/intel_cpu/src/nodes/executors/shl/shl_eltwise.cpp

```diff
@@ -0,0 +1,234 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shl_eltwise.hpp"
+#include "shl_utils.hpp"
+#include "csinn/csi_nn.h"
+#include "utils/debug_capabilities.h"
+
+namespace ov {
+namespace intel_cpu {
+
+inline void log_unsupported_prec(const std::vector<MemoryDescPtr>& srcDescs,
```

Comment:
This function seems to be unused. Could you please delete?

Reply:
Sure!